crates/cext/src/transpiler/passes/unitary_synthesis.rs

```diff
@@ -111,16 +111,18 @@ pub unsafe extern "C" fn qk_transpiler_pass_standalone_unitary_synthesis(
 #[cfg(all(test, not(miri)))]
 mod tests {
     use super::*;
-
-    use std::sync::Arc;
+    use pyo3::prelude::*;
```

Comment:
Introducing new uses of `pyo3` into the `cext` test suite certainly isn't ideal, but I think this will get removed again later?